packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -712,7 +732,7 @@ export default class Editor implements IEditor {
      */
     public getContentSearcherOfCursor(event?: PluginEvent): IPositionContentSearcher {
         return cacheGetEventData(event, 'ContentSearcher', () => {
-            let range = this.getSelectionRange();
+            let range = this.getSelectionRange() ?? new Range();

```

Comment:
Do we rather change the API for all content -traverser-returning methods to returning nullish values OR return empty content traverser (aka with a collapsed range)? If the latter, do we create an empty range on the content div?

Reply:
I'm thinking of creating an empty range on the firstChild of contentDiv. Can I assume there will always be one? 
```ts
let range = this.getSelectionRange() ?? createRange(this.core.contentDiv.firstChild!);
```

Reply:
No. If getSelectionRange() returns null, it means editor has never got focused before. So we don't know where should we focus to (and user doesn't know as well). So we should let it return null.